nautobot/extras/models/customfields.py

```diff
@@ -72,11 +72,11 @@ class ComputedField(BaseModel, ChangeLoggedModel, NotesMixin):
         help_text="Internal field name. Please use underscores rather than dashes in this key.",
         slugify_function=slugify_dashes_to_underscores,
     )
-    label = models.CharField(max_length=100, help_text="Name of the field as displayed to users")
-    description = models.CharField(max_length=200, blank=True)
+    label = models.CharField(max_length=255, help_text="Name of the field as displayed to users")
+    description = models.CharField(max_length=255, blank=True)
     template = models.TextField(max_length=500, help_text="Jinja2 template code for field value")
     fallback_value = models.CharField(
-        max_length=500,
```

Comment:
According to https://github.com/nautobot/nautobot/issues/2906#issuecomment-1912606173 we shouldn't be reducing the size of any fields.